src/cli/commands/autoresearch.ts

```
@@ -0,0 +1,658 @@
+    repoPath: string;
+    description: string;
+  }): Promise<void> {
+    const spinner = ora("Running experiment...").start();
+    try {
+      const repoPath = resolve(argv.repoPath);
+      const configPath = resolve(repoPath, ".autoresearch", "config.json");
+      if (!existsSync(configPath)) {
+        spinner.fail(chalk.red("No config. Run init first."));
+        process.exit(1);
+      }
+      let configRaw;
```

Biome (2.4.10)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)